packages/trader/package.json

```diff
@@ -83,6 +83,7 @@
     "webpack-node-externals": "^2.5.2"
   },
   "dependencies": {
+    "@deriv/api-types": "1.0.48",
     "@deriv/account": "^1.0.0",
     "@deriv/api-types": "1.0.48",
```

Comment:
"@deriv/api-types": "1.0.48" has already been added in line no.88

Reply:
Fixed